src/Kmd.Logic.Cvr.Client/CvrClient.DataDistributor.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using Kmd.Logic.Cvr.Client.Models;
+
+namespace Kmd.Logic.Cvr.Client
+{
+    public partial class CvrClient
+    {
+        /// <summary>
+        /// Creates Data Distributor CVR configuration.
+        /// </summary>
+        /// <param name="name">Name of the configuration.</param>
+        /// <param name="environment">Environment to use.</param>
+        /// <param name="certificate">Stream with certificate.</param>
+        /// <param name="certificatePassword">Password to the certificate.</param>
+        /// <returns>Created configuration.</returns>
+        /// <exception cref="ArgumentNullException">No parameter can be empty or null.</exception>
+        public async Task<CvrProviderConfiguration> CreateDataDistributorConfiguration(
+            string name,
+            ProviderEnviroment environment,
+            Stream certificate,
+            string certificatePassword)
+        {
+            if (string.IsNullOrEmpty(name)) throw new ArgumentNullException(nameof(name));
+            if (certificate is null) throw new ArgumentNullException(nameof(certificate));
+            if (string.IsNullOrEmpty(certificatePassword)) throw new ArgumentNullException(nameof(certificatePassword));
```

Comment:
I believe the password can be empty if the certificate does not require password?:) It can have default value even :)

Reply:
removed